docs/getting-started/intro-to-dockstore-tools-and-workflows.rst

```diff
@@ -1,27 +1,28 @@
-Introduction to Workflows, Tools, and Services
+Introduction to Workflows and Tools
 ==============================================
 
+Tools and workflows hosted on Dockstore are computational programs described in languages that make it easy for their reuse in different compute environments. Generally, descriptor languages leverage a containerized compute environment (such as a Docker image) that packages up all of the code and dependencies needed to complete the computational step.
+
+Not all descriptor languages make a distinction between a tool and a workflow. Most users will register workflows on Dockstore. 
+
 What is a workflow?
 -------------------
 
-In general terms, a workflow is a chain of commands that has several steps linked together, which create some sort of final output. For example, a simple two-step workflow that converts a VCF into a GDS file may conceptually look like this:
+In general terms, a workflow describes a chain of commands that may be executed in a specified order. For example, a simple two-step workflow that converts a VCF into a GDS file may conceptually look like this:
 
-vcf file --> convert to GDS file --> give each variant a unique ID --> a GDS file with unique IDs
+VCF file --> convert to GDS file --> give each variant a unique ID --> a GDS file with unique IDs
 
 In this case, the output of the conversion step is the input of the step that generates unique IDs, and the unique IDs step's output is the final output of the workflow: A GDS file with unique IDs. 
 
-Managing workflows can be complicated, as you may be chaining together many steps, each with different computational requirements. This is where workflow languages step in, allowing us to create formal "tasks" which can each have their own computational settings or Docker container, and chaining the output of one task into the input of another task.
+Managing workflows can be complex, as you may be chaining together many steps, each with different computational requirements. This is where workflow languages step in, allowing us to create formal "tasks" which can each have their own computational settings or Docker container, and chaining the output of one task into the input of another task.
 
 What is a tool?
 ---------------
-How Dockstore defines a tool is changing, but our aim is to align our tool definition with that of the languages we support.
-For the sake of simplicity, Dockstore only recommends registering a Dockstore tool when your work is a CWL CommandLineTool.
+A tool represents a single command line tool wrapped in a descriptor language.  Languages that formally describe tools (such as CWL) may chain them together into a workflow.
 
-If you want a deeper understanding of the history of Dockstore tools and where we plan on going, read :doc:`Dockstore Tools Overhaul </../advanced-topics/dockstore-tools-overhaul>`.
 
+Introduction to Services
```

Comment:
This comes out looking odd, IMHO:

![Screen Shot 2022-04-05 at 9 48 30 AM](https://user-images.githubusercontent.com/1049340/161806323-2241d025-b05c-4375-a6c0-ac99af1179c6.png)

Clicking on Introduction to Services in the left TOC takes you to a page whose top title is "Introduction to Workflows and Tools" (the services part is at the bottom, see screenshot).

I'm OK with less highlighting of services, since they are not very useful today. I think you should:

1. Just drop services from this page entirely
2. Drop this from this page, and either create a new page, or move this section to "Services" page
3. Restore them to this page

I lean towards 2.

Reply:
Ash and I agreed on option 2 and moved intro to Services page.